react-with-graphql-tutorial.md

```diff
@@ -307,11 +310,11 @@ That's it for the GraphQL query. You have learned everything about structuring y
 
```

Comment:
Should we Uppercase the exercises again?

Reply:
Yes.  Bulleted lists should only begin with lower case if they are part of the previous paragraph. Since the exercises are always their own paragraphs they should have capitals and periods like regular sentences.  I'll go through and make sure they're capitalized, like before.

Reply:
For sublists, I think lowercase works:
* Create several queries to request data from GitHub's GraphQL API using the following features:
  * objects and fields
  * nested objects
  * fragments
  * arguments and variables
  * operation names
  * directives

Reply:
Sounds good!